src/map/utils/battleutils.cpp

```diff
@@ -2297,6 +2297,12 @@ namespace battleutils
                 float sBlow2    = std::clamp((float)PAttacker->getMod(Mod::SUBTLE_BLOW_II), -50.0f, 50.0f);
                 float sBlowMult = ((100.0f - std::clamp(sBlow1 + sBlow2, -75.0f, 75.0f)) / 100.0f);
 
+                // Add subtle blow mods from NIN merits
+                if (PAttacker->objtype == TYPE_PC && PAttacker->GetMJob() == JOB_NIN && PAttacker->GetMLevel() > 74)
```

Comment:
I think this might make more sense to do in `charutils::CalculateStats` or `charutils::BuildingCharTraitsTable` where you can just add a subtle blow mod. `BuildingCharTraitsTable` already adds mods.

`GetMeritValue` already accounts for level >= 75 and job being NIN, so you could do something like..

```
        if (PChar->PMeritPoints)
        {
            PChar->addModifier(Mod::SUBTLE_BLOW, PChar->PMeritPoints->GetMeritValue(MERIT_SUBTLE_BLOW_EFFECT, PChar));
        }
 ```
Adding it deep in core means Lua won't be aware of the added subtle blow mod. In addition, this current PRed logic will go above the normal subtle blow 1 cap (is there evidence that the NIN subtle blow merit does this?)

Reply:
mod should be set in trait table and just having the trait should already handle adding the mod
https://github.com/LandSandBoat/server/blob/base/sql/traits.sql#L25
there's a reason the column is called "modifier" and this issue with caps is exactly it.

Reply:
bug, now known: non merit trait and merit trait won't stack because code for merit'ed traits will _delete_ non merited version of the trait before applying.

 (╯°□°)╯︵ ┻━┻